service-rpc/src/main/thrift/rpc.thrift

```diff
@@ -39,6 +39,9 @@ struct TSHandleIdentifier {
   // 16 byte secret generated by the server and used to verify that the handle is not being hijacked by another user.
   // In current version, it is not used.
   2: required binary secret,
+
+  // unique identifier in session. This is a ID to identify a query in one session.
+  3: required i64 queryId,
```

Comment:
Maybe it's time to merge TSHandleIdentifier and TSOperationHandle.

Reply:
TSHandleIdentifier is also used in TS_SessionHandle, so I think it may be better we still have it.